gradio/routes.py

```diff
@@ -56,30 +56,6 @@ def render(self, content: Any) -> bytes:
 ###########
 
 
-class PredictBody(BaseModel):
```

Comment:
Why did you remove these validations?

Better update them instead of removing.

Reply:
Good point, will update